app/views/contexts/calculated_summary_context.py

```diff
@@ -166,7 +172,7 @@ def _get_formatted_total(
         self,
         groups: list,
         current_location: Location,
-        calculation: Union[Callable, ImmutableDict],
+        calculation: Callable,
```

Comment:
Was this not correct?. It can be an Immutable dict or a callable

Reply:
👍